app/api/tickets.py

```diff
@@ -66,7 +68,18 @@ def query(self, view_kwargs):
         :param view_kwargs:
         :return:
         """
-        query_ = self.session.query(Ticket).filter_by(is_hidden=False)
+
+        if 'Authorization' in request.headers:
+            _jwt_required(current_app.config['JWT_DEFAULT_REALM'])
+            if current_user.is_super_admin or current_user.is_admin:
+                query_ = self.session.query(Ticket).filter_by()
```

Comment:
you can remove the `filter_by()`. Sorry noticed it just now :sweat_smile:

Reply:
Removed it. Thanks :smile: